src/main/java/server/constants/Status.java

```diff
@@ -0,0 +1,18 @@
+package server.constants;
+
+public enum Status {
+  OK("200 OK"),
```

Comment:
Good usage of enum! What else do you think could be enum in your code?

Reply:
I added a few more Enums (Method, Path, Header, Protocol) but I'm not sure if I went overboard with it and made unnecessary ones? I basically made one to store every string value that gets duplicated in the code -- is this correct usage?

Reply:
Correct, sometimes you can create a constant if you're going to use just it in one file/class. But if you're going to reuse these value over the code, enum will a better choice. Next time try not to add values that you don't use yet. Looks good to me!